ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_layers.py

```diff
@@ -336,6 +336,35 @@ def test_dropout1d(
         assert u.shape == v.shape == w.shape
 
 
+@handle_test(
+    fn_tree="functional.experimental.separable_conv2d",
```

Comment:
you are passing the wrong function tree here, and so the test fails at collection time. It should be `functional.ivy.experimental.separable_conv2d`.